mbed-os.lib

```diff
@@ -1 +1 @@
-https://github.com/ARMmbed/mbed-os/#64853b354fa188bfe8dbd51e78771213c7ed37f7
```

Comment:
I think we are not supposed to link to current master but to a released version. @0xc0170

Reply:
A few other examples do the same. Otherwise I will need to wait for Mbed 6.0.0-alpha 2 (to use c_lib)
https://github.com/ARMmbed/mbed-os-example-attestation/blob/master/mbed-os.lib
https://github.com/ARMmbed/mbed-os-example-tls/blob/master/authcrypt/mbed-os.lib
https://github.com/ARMmbed/mbed-os-example-wifi/blob/master/mbed-os.lib
etc.

Reply:
example master is dev, so can point to master sha (current workflow), will change update the release process